ui/apps/platform/src/Containers/NetworkGraph/utils/flowUtils.ts

```diff
@@ -67,7 +67,7 @@ export function getUniqueIdFromPeer(peer: Peer) {
     const direction = peer.ingress ? 'Ingress' : 'Egress';
     const { port } = peer;
     const { protocol } = peer;
-    const id = createUniqueFlowId({ entityId, direction, port, protocol });
+    const id = createUniqueFlowId({ entityId, direction, port: String(port), protocol });
```

Comment:
Interesting, I haven't seen a cast done in a function call before.

Reply:
As they say, "There's a first time for everything"